tensorflow_datasets/video/moving_sequence.py

```diff
@@ -0,0 +1,419 @@
+"""
```

Comment:
Let’s move this documentation into images_as_moving_sequence. Use MNIST instead of FashionMNIST as the example.

For the docstring here, you can put something simpler:

```
“””Turn simple static images into sequences of the originals bouncing around.

Adapted from <url to MovingMNIST page>.
”””
```